src/store/api/serializers.py

```diff
@@ -49,21 +73,20 @@ class ProductSerializer(serializers.Serializer):
     created_at = serializers.DateTimeField(read_only=True, format="%Y-%m-%d %H:%M")
     updated_at = serializers.DateTimeField(read_only=True, format="%Y-%m-%d %H:%M")
 
-    def validate(self, attrs):
-        """
-        Validates the product price after applying a discount to ensure it does not fall below its cost price.
-        """
-        cost_price = Decimal(attrs.get("cost_price", 0))
-        price = Decimal(attrs.get("price", 0))
-        discount = Decimal(attrs.get("discount", 0))
 
-        # Calculate the minimum acceptable price after discount
-        min_acceptable_price = cost_price * LOSS_FACTOR
-        discounted_price = price * (1 - discount / 100)
+class ProductPartialUpdateSerializer(BaseProductSerializer):
+    name = serializers.CharField(max_length=50, required=False)
+    category = serializers.CharField(read_only=True)
```

Comment:
I think it should be `category_id`, with `help_text` from which endpoint you can get it

Reply:
I added endpoint for categories list and change field "category" to the "category_id"  in this serializer